app/src/store/playgroundStore.tsx

```diff
@@ -130,12 +130,19 @@ export interface PlaygroundInstance {
   template: PlaygroundTemplate;
   tools: unknown;
   input: PlaygroundInput;
-  output: unknown;
+  output: ChatMessage[] | undefined | string;
   activeRunId: number | null;
   /**
    * Whether or not the playground instance is actively running or not
    **/
   isRunning: boolean;
+  /**
+   * Errors that occurred during parsing of initial playground data.
+   * For example, when coming from a span to the playground, the span may
+   * not have the correct attributes, or the attributes may be of the wrong shape.
+   * This field is used to store any issues encountered when parsing to display in the playground.
+   */
+  parsingErrors?: string[];
```

Comment:
I'm not sure we need to store this in state - can't we just pass it as props in the `SpanPlayground` and display them at the top of the page if needed?

Reply:
makes sense to me